src/document/json/rga_tree_list.ts

```diff
@@ -343,7 +343,7 @@ export class RGATreeList {
    */
   public delete(createdAt: TimeTicket, editedAt: TimeTicket): JSONElement {
     const node = this.nodeMapByCreatedAt.get(createdAt.toIDString());
-    if (node!.remove(editedAt)) {
+    if (!node!.isRemoved() && node!.remove(editedAt)) {
```

Comment:
In my opinion, it doesn't look good to use a non-null operator and logical not operator together. (`!node!`)

I think it would be better to change it as below. What do you think?
```ts
if (node!.isRemoved() === false && node!.remove(editedAt)) {
```